lib/radiator_web/graphql/admin/resolvers/editor.ex

```diff
@@ -221,6 +228,16 @@ defmodule RadiatorWeb.GraphQL.Admin.Resolvers.Editor do
     end
   end
 
+  def find_episode(_parent, %{guid: guid}, %{context: %{current_user: user}}) do
+    try do
+      with_episode user, guid, :get_episode_by_guid do
+        fn episode -> {:ok, episode} end
+      end
+    rescue
+      Ecto.MultipleResultsError -> {:error, "There is more than one episode with given guid"}
```

Comment:
See above. I'd rather narrow the list down to the ones the user has access to first. And then return. Also not sure if alex didn't want a public api here instead.